mkdocs.yml

```
@@ -0,0 +1,154 @@
+site_name: Automata
+site_description: Automata is a Python 3 library which implements the structures and algorithms for finite automata, pushdown automata, and Turing machines.
+site_url: https://automata.site.to.be.created.org
+repo_url: https://github.com/caleb531/automata
+repo_name: automata
+edit_uri: blob/main/docs/
+# author: Lewi Lie Uberg
+# multiple authors:
+authors:
+  - caleb531: https://github.com/caleb531
```

@eliotwrobson Can you link directly to my personal website here? https://calebevans.me/

```suggestion
  - caleb531: https://calebevans.me/
```